src/main/java/net/tirasa/connid/bundles/servicenow/dto/Resource.java

```diff
@@ -333,8 +334,9 @@ public class Resource implements BaseEntity {
     @JsonProperty("internal_integration_user")
     private String internalIntegrationUser;
 
+    @JsonDeserialize(using = SNComplexDeserializer.class)
     @JsonProperty("ldap_server")
-    private String ldapServer;
+    private SNComplex ldapServer;
```

Comment:
Q: I see that `SNComplex` is used for various attributes (`parent` for example) but `SNComplexDeserializer` is used only for `ldap_server`.

Doesn't make it sense to define `SNLdapServer` / `SNLdapServerDeserializer` ?

Reply:
Since all complex attributes have the same structure of `ldap_server`, probably would be better to apply that deserializer to all complex attributes.